cmd/load.go

```diff
@@ -13,6 +14,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 	"github.com/klauspost/compress/zstd"
+	"github.com/sethvargo/go-envconfig"
```

Comment:
I can imagine that as a CLI application you might want to have more freedom around where to load configs from. Env vars might be just one of sources.

In that case, I would think of something like:
- https://github.com/nil-go/konf
- https://github.com/ridgelines/go-config
- or even better - https://github.com/spf13/viper which extends the cobra lib you are already using.

So with these libs you would be able to seamlessly load configs from env vars/files of diff format/external sources/etc.